lib/util.py

```diff
@@ -0,0 +1,47 @@
+import imp
+import inspect
+import os
+import time
+
+def RateLimited(maxPerSecond):
+    minInterval = 1.0 / float(maxPerSecond)
+
+    def decorate(func):
+        lastTimeCalled = [0.0]
```

Comment:
Does this need to be a list for some reason?  Otherwise, looks good 👍

Reply:
That's copypasta from stackoverflow, and I asked the same question. Making it not a list gives some errors about initialization, so I left it in without looking any deeper.

Reply:
NIWID